charts/rucio-server/templates/deployment.yaml

```diff
@@ -86,6 +91,34 @@ spec:
         secret:
           secretName: {{ .Release.Name }}-server-cafile
 {{- end }}
+{{- end }}
+{{- if .Values.policyPackages.enabled }}
+      initContainers:
+        - name: install-policy-packages
+          # A slim Python image could be used instead, but using the same image as the base image for the main container ensures that the same Python version is used
+          image: almalinux:9
+          command:
+            - "sh"
```

Comment:
The use of an absolute pathname seems like a good practice.

Reply:
Could you clarify what you mean here?

Reply:
You can see that the [Containerfile](https://github.com/AlmaLinux/docker-images/blob/cbfe509d63b9d87c21c3f25a1d97d8f84f444cce/dockerfiles/al9/Dockerfile.default#L95) uses `/bin/bash` instead of just `bash`. Maybe we should follow that example.

Reply:
Addressed in latest revision